examples/30_extended/run_setup_tutorial.py

```diff
@@ -37,6 +37,11 @@
 import sklearn.ensemble
 import sklearn.impute
 import sklearn.preprocessing
```

Comment:
I don't think we need to import these separately as their used components get imported below?

Reply:
Done.